eZ/Publish/SPI/Persistence/Content/Handler.php

```diff
@@ -266,4 +266,15 @@ public function publish($contentId, $versionNo, MetadataUpdateStruct $metaDataUp
      * @param string $languageCode language code of the translation
      */
     public function removeTranslationFromContent($contentId, $languageCode);
+
+    /**
+     * Remove the specified Translation from the given Version (Draft) of a Content Object.
+     *
+     * @param int $contentId
+     * @param int $versionNo
+     * @param string $languageCode
+     *
+     * @return \eZ\Publish\SPI\Persistence\Content The Content Draft w/o removed Translation
+     */
+    public function deleteTranslationFromVersion($contentId, $versionNo, $languageCode);
```

Comment:
`public function removeTranslationFromContent($contentId, $languageCode);`
but
`public function deleteTranslationFromVersion($contentId, $versionNo, $languageCode);`

Wouldn't it be better to keep naming convention and use `remove` here as well?